src/lm_buddy/buddy.py

```diff
@@ -16,25 +20,52 @@ class LMBuddy:
     Simple wrapper around executable functions for tasks available in the library.
     """
 
-    def __init__(self, artifact_loader: ArtifactLoader = WandbArtifactLoader()):
-        self._artifact_loader = artifact_loader
+    # TODO: Store some configuration (e.g., tracking info, name) globally on the buddy
+    def __init__(self):
+        pass
+
+    def _generate_artifact_lineage(
```

Comment:
This is the central method that logs inputs to a run and logs output artifacts produced by a run.